app/forms/providers/means/regular_income_form.rb

```diff
@@ -0,0 +1,169 @@
+module Providers
+  module Means
+    class RegularIncomeForm
+      include ActiveModel::Model
+
+      INCOME_TYPES = %w[
```

Comment:
would using something like

```
def income_types
  TransactionType.not_children.credits.map(&:name)
   # OR
   transaction_type_options.map(&:name)
 end
```

... be neater and avoid duplicate sources of these types? might require testing changes though

Reply:
We could, but I hadn't yet.

The reason I've been reluctant to is because I don't feel like it necessarily makes sense for the attribute accessors and permitted params to depend explicitly on the `TransactionType` records.

I'm not 100% convinced `TransactionType` should be `ActiveRecord` models in the conventional sense —they feel more like "static" records to me. 

When it comes to the form rendered in the view, we do iterate the `TransactionType.not_children.credits` records though.

Let's revisit this if we run into it again (i.e we add a new transaction type and we create the record, but forget to add attribute accessors or permit the new parameters).

How does that sound?